libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common.xcconfig

```diff
@@ -15,3 +15,6 @@ LIBRARY_SEARCH_PATHS = $(inherited)
 
 // Make sure to use only API that are safe for application extensions
 APPLICATION_EXTENSION_API_ONLY = YES
+
+MODULEMAP_FILE = $(SRCROOT)/SalesforceSDKCore/SalesforceSDKCore.modulemap
```

Comment:
Fair enough, I didn't see the -Dynamic file there when I added this.  Admittedly, there's no harm in assigning a module map definition to a static target since Xcode will ignore it, but for accuracy, you're right.